### ISSUE TITLE
Map SDG tag terms to OBO Library terms

### ISSUE THREAD

**Issue by rlwalls2008** [SDG-InterfaceOntology/sdgio#7]
Chris, can you run an automatic search to map the terms in the first column of this file (https://github.com/SDG-InterfaceOntology/sdgio/blob/master/docs/mappings/SDG_tag_words_Gemet_OBO.csv) to OBO library terms?

**Comment by cmungall** [comment]
Isn't this covered by #4?

**Comment by rlwalls2008** [comment]
Related, but there are a lot of terms in here that aren't in GEMET. Not sure how many of those will be in OBO Library either, though. May not be worth the effort.

**Comment by cmungall** [comment]
Ah yes I see the GEMET column

**Comment by cmungall** [comment]
c15: I think adult should be hsapdv, it means human adult presumably

**Comment by rlwalls2008** [comment]
Agreed. I spent about 15 minutes on this mapping, so there are going to be some changes as we do it more carefully.
